### DISCUSSION TITLE
How can I restore GitHub contribution calendar updates?

### DISCUSSION THREAD

**Discussion by GustavoPellanda** [community/community#122579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm currently working on a project and regularly pushing updates to my personal GitHub profile. Initially, my contributions were being reflected on my calendar, as expected. However, I recently left an organization with which I was no longer actively involved. Since then, my pushes to repositories created before leaving the organization have ceased to appear on my contribution calendar.

Some notes:
- I can confirm that the email associated with the Git account I'm using for pushes matches the one on my GitHub profile.
- Previous pushes I made to repositories  before departing the organization still show up on my profile.
- Pushes I make to repositories created after leaving the organization are reflected on the calendar.
- I made no alterations to my profile other than leaving the organization.
- The repository I'm pushing to is public.
- I'm the owner of the repository and it is not related to the organization I've left.
- I've given github sufficient time to update to make sure the contributions actually won't show on the calendar.

All previous answers to this issue I've found were: checking the email addresses, turning on private contributions to the calendar, giving github time to update or about leaving an organization invalidating contributions. None of those help in my case.

I want to be able to see my contributions on the calendar like I was before.

**Comment by afifurrohman-id** [comment]
> I want to be able to see my contributions on the calendar like I was before

If you just worry about green graph, then try this command to commit in any of your repository and push it:
```sh
git commit -m "msg" --date=<your-missing-date> # e.g 2023-02-13
```

And you will see that your missing contribution date will appear in contribution graph.

**Reply by GustavoPellanda** [reply]
I appreciate the response. However, using the --date flag to make commits on past dates is also not causing the green squares to appear on the calendar. Absolutely no commits I make to the repository I created before leaving the organization are showing up anymore.

**Comment by msi-end** [comment]
* **Check Commit Timestamps:** Ensure that the timestamps of your commits are within the current calendar year. GitHub's contribution calendar only displays contributions made within the past year. If your commits have older timestamps, they won't appear on the calendar.

* **Verify Commit Author Information:** Double-check the author information of your commits to confirm that they match the email associated with your GitHub account. Even though you've checked the email address, it's worth verifying this detail again, especially if you're using multiple email addresses for your commits.

* **Review Repository Settings:** Check the settings of the repository where you're pushing your updates. Make sure that the repository visibility is set to "Public" and that you have the necessary permissions to push commits to the repository.

* **Check GitHub Status:** Sometimes, GitHub experiences issues or delays in updating contribution data. You can check the GitHub Status page ([https://www.githubstatus.com/](https://www.githubstatus.com/)) to see if there are any ongoing incidents that might be affecting contribution tracking.

* **Reach Out to GitHub Support:** If you've exhausted all other options and your contributions still aren't appearing on your calendar, consider reaching out to GitHub Support for further assistance. They may be able to investigate the issue on their end and provide additional insights or solutions.

**Reply by GustavoPellanda** [reply]
Timestamps are within the current calendar year. Checking author information, turning the repository public and giving github time to update, as I've stated, aren't applicable to my case.

**Comment by BaselAbuHamed** [comment]
git commit -m "msg" --date=<your-missing-date> # e.g 2024-03-13
